pkgs/games/super-tux-kart/default.nix

```
@@ -34,6 +34,14 @@ in stdenv.mkDerivation rec {
     curl fribidi bluez libjpeg libpng
   ];
 
+  # https://github.com/supertuxkart/stk-code/issues/3557#issuecomment-440794379
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/supertuxkart/stk-code/commit/3a3953f38c3555e87f3608d0291dbfccf34e9775.patch";
+      sha256 = "04kyq3dgfrvai4favn759qh4f3i9fkcsbav0y6hbl6cx73658rfd";
```

According to the build log the hash should be:
```suggestion
      sha256 = "13mr5pwf45g7frpxzlxiyiq39qi3z9spd7l6gi8i3dr3li9wrb1k";
```

---

Right... Sorry about that. Thanks!